### ISSUE TITLE
Unable to expand accordion using the keyboard with NVDA running on Chromium-based browsers

### ISSUE THREAD

**Issue by dandalpiaz** [web-illinois/ilw-accordion#21]
When trying to use accordions (v1.3) with NVDA running with a Chromium-based browser (Chrome, Edge, etc.), I can't seem to expand or collapse the accordions using the 'enter' or 'space' key. Tested here, https://builder3.toolkit.illinois.edu/preview/ilw-accordion/1-3/index.html .

If I force NVDA into 'focus' mode, I can get them to work, but I've never had to do that for other buttons before - using the enter or space key in 'browse' mode always works as expected. 

I tried recreating the issue with a simple ARIA button and a typical 'onclick' event, thinking the ARIA button was maybe throwing things off, but that seemed to work fine. I wonder if Lit's click/keydown event handlers are somehow conflicting with something NVDA is doing.

**Comment by bryanjonker-illinois** [comment]
From Tim: 

Good morning, Bryan, 

The ILW Accordion panels are not operating as expected. On Sociology's website, on the “Faculty Research Clusters” page, under the Research tab, https://sociology.illinois.edu/research/faculty-research-clusters, when using NVDA in the default Browse mode, the panels will not respond to the keyboard prompt to open and close. The user has to switch to Focus mode to get the panels to respond to the keyboard. This is not expected behavior.
